README.md

```diff
@@ -3,37 +3,121 @@
 ## Objectives
 
 1. Explain why we use spies in JavaScript
-2. Describe how to use Jasmine spies
-3. Practice using Jasmine spies
+2. Describe how to use Sinon spies
```

Comment:
Thanks for switching this -- I had this idea that we'd try to stick with Jasmine since there was already content, but Sinon is 👍 for sure